### DISCUSSION TITLE
XDrip broadcast unreliable. (Libre3)

### DISCUSSION THREAD

**Discussion by elgrupo123** [NightscoutFoundation/xDrip#3040] [Bug reports] [answered]
XDrip broadcast unreliable.
The XDrip Broadcast works very unreliably for me. It doesn't matter if it's a 1 minute or 5 minute interval. After about 30-50 minutes it breaks off and no values arrive in AAPS. But the values are still displayed in the XDrip. The issue seems to be known https://github.com/blaqone/xDrip

But even this modified version doesn't fix the error for me.
With Juggluco (XDrip Broadcast) there are no problems. The values are reliably forwarded to AAPS without interruptions
directed.
Anyone an idea??

![PXL_20230824_104503252](https://github.com/NightscoutFoundation/xDrip/assets/92778761/41c9aaa4-9965-40b8-9f31-caa394cddbb5)

**Comment by jamorham** [comment, marked as answer]
Change the noise blocking level back to the default which is titled `Only block noise level extreme` At the moment you are blocking anything above 10. This is in `Settings` -> `Inter-app settings`

**Reply by elgrupo123** [reply]
My setting was "Block High noise and worse". It is currently running without interruptions. But it is still the modified version. Can I switch to the normal Nightly without hesitation, or rather not?

**Reply by jamorham** [reply]
I'm not aware of any problems with broadcast to AAPS, but your setting will cause data to not be sent when noise is > 10. There is no fault with the software it is doing exactly what you have instructed it to do by changing that setting. It is "working now" because your noise level is below that threshold. If you want to ensure all data is sent to AAPS then change that setting to be more tolerant of noise.

**Reply by elgrupo123** [reply]
It also works with the nightly version. Excellent.
Thank you Jamorham!!!

**Comment by jamorham** [comment]
If that doesn't solve the issue then also look at `Identify Receiver` and check that this has the AAPS package name in there.
